Improve dtt pattern docs (#789)
astronomer/astro-sdk: python-sdk/docs/astro/files/operators/get_file_list.rst

The supported filesystems are :ref:`file_location`

.. warning::

>     Fetching a lots of files using this method can lead to overload XCOM and also can create lots of parallel tasks if using in dynamic task map expand method.

Reviewer: ```suggestion
    Fetching a lot of files using this method can lead to overloaded XCOM. This can create lot of parallel tasks when used in dynamic task map ``expand`` method.
```
